### ISSUE TITLE
Zip::Error when downloading SignatureRequest documents by calling signature_request_files

### ISSUE THREAD

**Issue by bradpurchase** [hellosign/hellosign-ruby-sdk#93]
Hi there,

My team updated the `hellosign-ruby-sdk` gem in our bundle from `3.6.4` to `3.7.1` and started experiencing the following error each time a `signature_request_all_signed` callback was received:

```
Zip::Error Zip end of central directory signature not found
```

When checking out the files that are downloaded, what we get is a corrupted .zip file, causing the error above.

It is unclear to me whether there is anything we need to change to get this working again between these two versions, but it's consistent in all of our environments. We have downgraded the gem to `3.6.4` for now.

**Comment by jyoung488** [comment]
Hi @bradpurchase - Thanks for notifying us! We are looking into this and hope to resolve it soon!

**Comment by jyoung488** [comment]
Fixed in #94 !

**Comment by bradpurchase** [comment]
@jyoung488 thanks for the quick response! We will update our gem to `3.7.2` and try again 😁
